bitnami/harbor/values.yaml

```
@@ -1249,6 +1261,10 @@ jobservice:
   ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
   ##
   affinity: {}
+  ## @param jobService.priorityClassName Priority Class Name
```

The metadata is case-sensitive, please fix this minor issue:
```suggestion
  ## @param jobservice.priorityClassName Priority Class Name
```
This will fix the failing test.